src/jsonschema.rs

```diff
@@ -85,12 +87,17 @@ pub struct Tag {
 
 impl Tag {
     fn get_type(&self) -> Type {
-        match &self.data_type {
-            Value::String(string) => {
+        match (&self.data_type, &self.format) {
+            (Value::String(string), Some(format_str))
+                 if string == "string" && format_str == "date-time" =>
+             {
+                 Type::Atom(Atom::DateTime)
```

Comment:
Kept this as a separate, explicit `DateTime` type to make downstream transpiling more clear.

Reply:
If this in response to a `Format::DateTime` type, it's fine to map between `Format::DateTime` and `Atom::DateTime`. `serde` will handle mapping the string to the rust type, and the pattern matching will be nicer. It'll look like:

```rust
(Value::String(string), Some(Format::DateTime)) => { ... }
```

Reply:
I _think_ we do want to verify that the `string` value is `"string"`, or else that doesn't match to a datetime type.